package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "asq-multi-choice-q",
-  "version": "0.2.0",
+  "version": "0.2.1",
   "description": "Multi-choice question type for ASQ.",
-  "author": "Zhenfei Nie <zhen.fei.nie@usi.ch>, Vassilis Triglianos <triglian@gmail.com>",
+  "author": "Zhenfei Nie <zhen.fei.nie@usi.ch>, Vassilis Triglianos <triglian@gmail.com>, Gustavo Graziani <gustavo.graz@outlook.com>",
```

Comment:
one author, multiple contributors per https://docs.npmjs.com/files/package.json#people-fields-author-contributors

See here for an example: 

http://browsenpm.org/package.json